ZenPacks/zenoss/HttpMonitor/http.py

```diff
@@ -25,11 +25,36 @@
 
 log = logging.getLogger('zen.HttpMonitor')
 
+class RedirectAgentZ(RedirectAgent):
+    def __init__(self, agent, onRedirect="ok", port=80, proxy=""):
```

Comment:
I don't think you need default arguments here because all arguments are supplied by the caller.  Besides, the default argument for `onRedirect` is wrong: "ok" isn't a supported by `RedirectAgentZ`.